test/services/stake_pools/calculate_average_stats_test.rb

```diff
@@ -67,7 +67,7 @@ class CalculateAverageStatsTest < ActiveSupport::TestCase
         stake_pool: stake_pool,
         delegated_vote_account_address: "vote_acc",
         validator: validator,
-        active_stake: 422_555_666
+        active_stake: 2_555_666_777
```

Comment:
What the reason for such a drastic value change?

Reply:
This way the delinquent_count is covered in test, since stake_account has more than 1 SOL and score is delinquent. As result the validator is considered as a delinquent.